cmd/resim/commands/batch.go

```diff
@@ -346,6 +351,93 @@ func createBatch(ccmd *cobra.Command, args []string) {
 	}
 }
 
+func batchToSlackWebhookPayload(batch *api.Batch) *slack.WebhookMessage {
+	var (
+		suite   *api.TestSuite
+		system  *api.System
+		reports *[]api.Report
+	)
```

Comment:
```suggestion
	var (
		suite   *api.TestSuite
		system  *api.System
		reports *[]api.Report
	)
```
We generally declare and assign in golang in a lot of cases (see below)